improtreskApi/settings.py

```diff
@@ -29,11 +24,12 @@
     AWS_SECRET_ACCESS_KEY = os.environ.get('DJANGO_AWS_SECRET_ACCESS_KEY')
     AWS_STORAGE_BUCKET_NAME = os.environ.get('DJANGO_AWS_STORAGE_BUCKET_NAME')
 
-# SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = os.environ.get('DJANGO_DEBUG', False)
```

Comment:
OK, good enough solution, if it is only variable, that we will need to set. But please add this command at least to README.